observable-pt1/src/app/app.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { MyService } from './my-service.service';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent implements OnInit {
+  title = 'observable-pt1';
+  users: Observable<Array<Object>>;
```

Comment:
daca doresti sa folosesti Object va trebui sa pui [] in loc de Object
users: Observable<Array<[]>>;